### PR TITLE
Update Copyright to 2022

### PR DESCRIPTION
**Pull request by Daniel-Fan**

### CODE REVIEW

File: controllers/operandconfig.go [hunk 1]
```diff
@@ -0,0 +1,469 @@
+//
```
Comment (horis233) [inline controllers/operandconfig.go:1]: Why this new file is created?

Comment (Daniel-Fan) [reply controllers/operandconfig.go]: Yeah, I think it is introduced when I rebased some commits.
Updated~

**Review by horis233 — APPROVED** [review]
/lgtm

**Pull request merged**